cmd/logger.go

```diff
@@ -160,10 +196,20 @@ func logIf(level logrus.Level, source string, err error, msg string, data ...int
 		"cause":  err.Error(),
 	}
 
+	var stack []string
 	if terr, ok := err.(*errors.Error); ok {
 		fields["stack"] = strings.Join(terr.Stack(), " ")
```

Comment:
Rather than calling `terr.Stack()` multiple times can we call it once in this `if` conditional? 

```go
var stack []string
if terr, ok := err.(*errors.Error); ok {
         stack = terr.Stack()
}

if len(stack) > 0 {
       fields["stack"] = strings.Join(terr.Stack(), " ")
}
```